webclient/app/src/app/commons/validatedTextField/SimpleValidatedTextField.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { useEffect, useState } from "react";
+import {TextField} from "@mui/material";
+import PropTypes from "prop-types";
+
+function ValidatedTextField(props) {
```

Comment:
What is the difference to the other ValidatedTextField?

Reply:
I had some problems displaying the error messages at the right time and won't override ValidationTextField because it is used somewhere else. I already created a ticket to combine the implementations (ab#49).